tests/auth/managers/fab/test_fab_auth_manager.py

```diff
@@ -81,3 +81,14 @@ def test_get_url_login_with_next(self, mock_url_for, auth_manager):
         auth_manager.security_manager.auth_view.endpoint = "test_endpoint"
         auth_manager.get_url_login(next_url="next_url")
         mock_url_for.assert_called_once_with("test_endpoint.login", next="next_url")
+
+    def test_get_url_logout_when_auth_view_not_defined(self, auth_manager):
+        with pytest.raises(AirflowException, match="`auth_view` not defined in the security manager."):
+            auth_manager.get_url_logout()
```

Comment:
Might maybe be worth mocking url_for() and asserting that it doesn't get called, but I'm fine with this as it is.